taqueria-plugin-flextesa/index.ts

```diff
@@ -49,6 +49,57 @@ Plugin.create(_i18n => ({
             ],
             encoding: "json"
         }),
+        Task.create({
+            task: "typecheck",
+            command: "typecheck <sandboxName> [sourceFiles...]",
+            aliases: ["tc"],
+            description: "Typecheck Michelson contracts",
+            options: [],
+            handler: "proxy",
+            positionals: [
+                PositionalArg.create({
+                    placeholder: "sandboxName",
+                    description: "The name of the sandbox to use"
+                }),
+                PositionalArg.create({
+                    placeholder: "sourceFiles",
+                    description: "The names of the Michelson contracts you wish to typecheck, separated by space"
+                }),
+            ],
+            encoding: "json"
+        }),
+        Task.create({
```

Comment:
Currently, the output from a failed command does not show the user what short flags/arguments are missing

The current output is: 
```
taqueria/333 on  557-typechecker-task-JCC 
❯ taq simulate local taco.tz
taq simulate <sandboxName> <sourceFile>

Run Michelson contracts as a simulation

Positionals:
  sandboxName  The name of the sandbox to use                         [required]
  sourceFile   The name of the Michelson contract you wish to simulate, separate
               d by space                                             [required]

Options:
      --version     Show version number                                [boolean]
      --build       Display build information about the current version[boolean]
  -p, --projectDir  Path to your project directory               [default: "./"]
      --configDir   Config directory (default ./.taq)        [default: "./.taq"]
  -e, --env         Specify an environment configuration
      --help        Show help                                          [boolean]
  -s, --storage     The initial storage used to run the script        [required]
  -i, --input       The input used to run the script                  [required]

Missing required arguments: storage, input
```

The output for the command should include the short flag options as part of the structure of the command, and details below so a user immediately understand what is required when the task fails. Given that there are 4 required arguments for the command, we should not expect the user to remember them.

Something along the lines of::
```
❯ taq simulate 
taq simulate sandboxName sourceFile -s '{initial storage: 1}' -i 'format of input here'

Run Michelson contracts as a simulation

Positionals:
  sandboxName  The name of the sandbox to use                         [required]
  sourceFile   The name of the Michelson contract you wish to simulate, separate
               d by space                                             [required]

Required Parameters:
Initial storage: -s        Describe format for initial storage here
Input: -i                        Desicrbe format for input here
```

Reply:
I think this UI improvement might be outside of the scope of this project. It works like this in every other plugin. e.g. `taq compile --help`

Reply:
That is another case where it's optional (can be run with or without `--help`)

If it's an architectural limitation, could we not also use positional arguments for initial storage and input?

Reply:
I don't think it makes sense to use propositional arguments for storage and input. They are, by nature, options. Propositionals are CLI or plugin related, but Options are task-related, as in they are meant to be passed to some underlying process that the tasks invokes

Reply:
The bottom of the error output, `Missing required arguments: storage, input`, prompts the user to specify them.

Reply:
👍

Reply:
Can we at least make the error output at the bottom indicate the structure of the command, or what parameters are missing? (taq simulate sandboxName sourceFiles --storage {} --input {})?